dev-tools/packaging/templates/docker/Dockerfile.elastic-agent.tmpl

```
@@ -30,8 +30,27 @@ FROM {{ .from }}
 {{- if contains .from "ubi-minimal" }}
 RUN for iter in {1..10}; do microdnf update -y && microdnf install -y shadow-utils jq &&  microdnf clean all && exit_code=0 && break || exit_code=$? && echo "microdnf error: retry $iter in 10s" && sleep 10; done; (exit $exit_code)
 {{- else }}
+
+RUN set -e ; \
+  case "$(arch)" in \
+    aarch64) \
+        for iter in {1..10}; do yum update --setopt=tsflags=nodocs -y \
+            && yum install --setopt=tsflags=nodocs -y http://mirror.centos.org/centos/7/updates/aarch64/Packages/bind-license-9.11.4-26.P2.el7_9.5.noarch.rpm \
+            && yum clean all \
+            && yum --enablerepo=updates clean metadata \
+            && exit_code=0 && break || exit_code=$? && echo "yum error: retry $iter in 10s" && sleep 10; done; (exit $exit_code) ; \
+        ;; \
+    *) \
+        ;; \
+  esac ;
+
 # Installing jq needs to be installed after epel-release and cannot be in the same yum install command.
-RUN for iter in {1..10}; do yum update --setopt=tsflags=nodocs -y && yum install --setopt=tsflags=nodocs -y epel-release &&  yum clean all && exit_code=0 && break || exit_code=$? && echo "yum error: retry $iter in 10s" && sleep 10; done; (exit $exit_code)
+RUN for iter in {1..10}; do yum update --setopt=tsflags=nodocs -y \\
+    && yum install --setopt=tsflags=nodocs -y epel-release \\
+    && yum clean all \\
+    && yum clean metadata \\
+    && exit_code=0 && break || exit_code=$? && echo "yum error: retry $iter in 10s" && sleep 10; \\
+done; (exit $exit_code)
```

Previous CI build failed building the elastic-agent docker image. Are these double back-slashes needed?
```suggestion
RUN for iter in {1..10}; do yum update --setopt=tsflags=nodocs -y \
    && yum install --setopt=tsflags=nodocs -y epel-release \
    && yum clean all \
    && yum clean metadata \
    && exit_code=0 && break || exit_code=$? && echo "yum error: retry $iter in 10s" && sleep 10; \
done; (exit $exit_code)
```

---

This was the failure I am refering to:
```
[2021-05-11T19:38:42.313Z] Error: failed building elastic-agent type=docker for platform=linux/amd64: failed to build docker: running "docker build -t docker.elastic.co/beats/elastic-agent:8.0.0-SNAPSHOT build/package/elastic-agent/elastic-agent-linux-amd64.docker/docker-build" failed with exit code 1
```

---

No, they do not. However, we never get to this point, so I did not bother to clean this up yet.